test/allof/resources/enterYourNameExpectedContent.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
Move the `resources` dir to the `test` and rename it `fixtures` to align with our other repos.
Have the `allOf` dir (currently the parent of the `resources` dir) be the child of the new `fixtures` dir.

So in other words, change `/test/allof/resources/` to `/test/fixtures/allOf`